src/tests/load-function-module.test.ts

```diff
@@ -31,6 +31,21 @@ Deno.test("LoadFunctionModule function", async (t) => {
     );
   });
 
+  await t.step("should load the js file if ts file is invalid", async () => {
+    const tsModule = await LoadFunctionModule(
+      [
+        `${functionsDir}/badFile.ts`,
+        `${functionsDir}/wacky.js`,
+      ],
+    );
+    assertExists(tsModule);
+    assertEquals(
+      tsModule.default.name,
+      "wackyJS",
+      "typescript file invalid, js file loaded",
```

Comment:
This should be a [Custom Error Message](https://api.slack.com/methods/functions.list/test) in case this assertion doesn't work. Right now this reads like a success message.